### DISCUSSION TITLE
Set Order for Filters

### DISCUSSION THREAD

**Discussion by StanBarrows** [laravel/nova-issues#4680] [Q&A] [answered]
Heyhey

I have a diverse set of filters. Some were created manually, and some were from filterable() Nova Fields. 

- How can I push a custom filter down in priority?
- Define the order of filters.

<img width="402" alt="CleanShot 2022-08-10 at 09 09 44@2x" src="https://user-images.githubusercontent.com/10268813/183837646-94b80e1e-72d4-4724-978c-e59a0c80c11d.png">

Cheers
Stan

**Comment by crynobone** [comment, marked as answer]
You can override `availableFilters()` method on the Resource class, see `Laravel\Nova\ResolvesFilters` trait for the usage reference.

**Reply by StanBarrows** [reply]
Thx a lot!
